Create nl_be.json

EnsoftCo opened the pull request:
Adding Belgian Dutch

I hope this is OK, I'm very new to this.
I've pulled this request instead of doing this locally for 2 reasons:
1. Hoping to help other Dutch speaking people
2. I'm running Docker in Libreelec and I've read that modifying a container is not good practice so I'm hoping as this get's merged the file will be present in the docker image so I don't have to mod my docker container???

Any tips, tricks, remarks or help is welcome :)

keydon commented:
Hey there, thank you for your contribution.
I would gladly merge your nl_be.json file.
But you also added a redundant github-docker-workflow that I dont understand, why you added it to this pull-request.

And btw. its not bad practice to take existing docker images and modifing them to your needs. But from your modification more ppl can benefit, so thank your for this pull request.

EnsoftCo commented:
> Hey there, thank you for your contribution.
> I would gladly merge your nl_be.json file.
> But you also added a redundant github-docker-workflow that I dont understand, why you added it to this pull-request.
> 
> And btw. its not bad practice to take existing docker images and modifing them to your needs. But from your modification more ppl can benefit, so thank your for this pull request.

Hi,

I added the docker workflow to be able to be able to add my own branch to docker.
Perhaps this was not the right way to do it?

I did it, to able to test out my branch. Once my version gets merged it is of course no longer needed.

If I delete my docker workflow, can you merge my dutch file?

keydon commented:
Our docker workflow file is parameterized with three secrets, one for each (username, password, registry-imagename) So you should be able to use our workflow from within your repository
